ush/gefs_atmos_prep.sh

```diff
@@ -1,154 +1,176 @@
-#!/bin/bash
+#! /usr/bin/env bash
 
-echo "$(date -u) begin $(basename $BASH_SOURCE)"
-export PS4="${PS4}${1}: "
+source "${HOMEgfs:-${HOMEgefs}}/ush/preamble.sh"
 
-set -xa
-if [[ ${STRICT:-NO} == "YES" ]]; then
-	# Turn on strict bash error checking
-	set -eu
-fi
+#echo "$(date -u) begin $(basename $BASH_SOURCE)"
+#export PS4="${PS4}${1}: "
+
+#set -xa
+#if [[ ${STRICT:-NO} == "YES" ]]; then
+#  # Turn on strict bash error checking
+#  set -eu
+#fi
 
 export mem=$1
 export nmem=$(echo $mem|cut -c 2-)
 nmem=${nmem#0}
 
-export INIDIR=$DATA
-export OUTDIR=$GESOUT/enkf/$mem
-INITDIR=$GESOUT/init/$mem
-mkdir -p $INIDIR
-mkdir -p $OUTDIR
-mkdir -p $INITDIR
+YMD=${PDY} HH=${cyc} MEMDIR=${mem} generate_com -x COM_ATMOS_INPUT
+
+#export INIDIR=$DATA
+export OUTDIR=${COM_ATMOS_INPUT} # ${COMOUT}/$mem/atmos/INPUT
```

Comment:
`generate_com` should be called at the j-job level and be read-only whenever possible. For this job, that will mean also pulling up the `mem` (and `memchar`, see correction) definition up to the j-job. Also, this should be:

```suggestion
YMD=${PDY} HH=${cyc} MEMDIR=${memchar} generate_com -rx OUTDIR:COM_ATMOS_INPUT_TMPL
```

In GFS, I replaced all these indirect variable names (like `INDIR`, `OUTDIR`) with the actual location (`COM_ATMOS_INPUT`), but we can skip that for now until we are sure my effort there isn't rejected.